### ISSUE TITLE
Using Termux-X11

### ISSUE THREAD

**Issue by twaik** [AndronixApp/AndronixOrigin#51]
Hi. There is an abandoned project https://github.com/termux/termux-x11 . You can try to fix it to use inside your project. It is much faster than VNC. It is impossible to use termux-x11 with xwayland inside termux app home dir so you will need to have patched xwayland inside andronix app.

**Comment by ultrahacx** [comment]
Sorry to say so but we are not going to add any of the maintained or unmaintained X11 framebuffer addons as of now. Termux-x11 is already abandoned, plus it is not intended to work with PRoot as it does with regular termux environment. We would have given a thought to XSDL but its not compatible on all devices(including mine since last 2 months) and does not have a very pleasing UX/UI design as of now. If the developer maintains the app on regular basis with some UI improvements then we will be more than happy to integrate it with our system. 

In conclusion we are not going to move from the regular VNC system as of now.

**Comment by twaik** [comment]
Termux-x11 works with package-local (in-app) xwayland. And it works on the latest devices. Using non-framebuffer Xorg implementation for now is impoasible in Android. Give it a try.

**Comment by JacenBridger** [comment]
Hello, termux x-11 is online i.e, it is progressing. Any chance being added to modded os? Thank You.

**Comment by Abdullrahman-Dawaliby** [comment]
Please I installed ubuntu 22.04 and did everything right but vnc has too bad screen as colors so please tell us how to integrate x11 to run with x11. Sorry for bad English

**Comment by twaik** [comment]
It seems like you can review this issue one more time. Termux:X11 is not abandoned anymore.

**Comment by victorwads** [comment]
It works guys... Flawless

**Comment by AltyFox** [comment]
@victorwads I can't figure out how to get it to work. Any pointers?
